src/toil/lib/misc.py

```diff
@@ -77,9 +77,13 @@ def slow_down(seconds: float) -> float:
 
     return max(seconds, sys.float_info.epsilon)
 
-def printq(msg: str, quiet: bool) -> None:
+
+def printq(msg: str, quiet: bool, log: bool = False) -> None:
     if not quiet:
-        print(msg)
+        if not log:
+            print(msg)
+        else:
+            logger.debug(msg)
```

Comment:
The new `log` feature here doesn't seem to get used.

If this function starts to grow options, it probably should have a docstring, maybe with a nice example of how using it is succinct.